lib/usage.js

```diff
@@ -122,9 +134,10 @@ module.exports = function (yargs, y18n) {
         return acc
       }, {})
     )
+    var theWrap = getWrap()
```

Comment:
To avoid redundant calls to `getWrap()` in the private `maxWidth` function, we could just pass `theWrap` to it as an additional argument.